packages/@divvi/mobile/locales/base/translation.json

```diff
@@ -2853,7 +2853,18 @@
   "reviewTransaction": {
     "title": "Review Send",
     "totalPlusFees": "Total Plus Fees",
-    "multipleTokensWithPlusSign": "≈ {{amount1}} {{symbol1}} + {{amount2}} {{symbol2}}"
+    "multipleTokensWithPlusSign": "≈ {{amount1}} {{symbol1}} + {{amount2}} {{symbol2}}",
+    "totalBottomSheet": {
+      "totalPlusFees": "Total Plus Fees",
```

Comment:
do we need this to be duplicated with the one 3 lines above?

i'm also wondering if we should add the estimated fee and max fee labels to the highest level of the translation json, so that it can be reused everywhere since the goal of this work is to introduce consistency. i don't feel strongly about this but it does make me sad to see us introducing the same type of translations time and time again, giving the ability to have different variations in each place